gcp/README.md

```diff
@@ -673,3 +673,55 @@ resolved/update is done.
 If this downtime will be disruptive to your work ("We're giving a demo for the
 Prime Minister at that time!"), please let me know immediately.
 ```
+
+#### External backups
+
+##### Setup.
```

Comment:
Titles are not sentences so no ending period. Most of the new titles have this bug.

Reply:
It has been removed from the README https://github.com/gpii-ops/gpii-infra/commit/38542c9230965b1c36279bdf603dbb40192b1a3e